docs/cookbooks/multus/README.md

```diff
@@ -0,0 +1,339 @@
+cd antrea
+cp docs/cookbooks/multus/test/Vagrantfile test/e2e/infra/vagrant/
+cd test/e2e/infra/vagrant
+./infra/vagrant/provision.sh
+```
+
+The last command will take around 10 to 15 minutes to complete. After that, your
+cluster is ready and you can set the `KUBECONFIG` environment variable in order
+to use `kubectl`:
+
+```bash
+export KUBECONFIG=`pwd`/infra/vagrant/playbook/kube/config
+kubectl cluster-info
+```
+
+The cluster that you have created by following these steps is the one we will
+use as an example in this guide.
+
+## Practical steps
+
+### Step 1: Deploying Antrea
+
+For detailed information on the Antrea requirements and instructions on how to
+deploy Antrea, please refer to
+[getting-started.md](/docs/getting-started.md). To deploy the latest version of
+Antrea, use:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea.yml
+```
+
+You may also choose a [released Antrea
+version](https://github.com/vmware-tanzu/antrea/releases).
+
+### Step 2: Deploy Multus as a DaemonSet
+
+```bash
+git clone https://github.com/intel/multus-cni.git && cd multus-cni
+cat ./images/multus-daemonset.yml | kubectl apply -f -
+```
+
+### Step 3: Create a NetworkAttachmentDefinition
+
+Make sure that you are using the correct interface name for `"master"`: it
+should be the name of the Node's network interface (e.g. `eth0`) that you want
+to use as the parent interface for the macvlan secondary Pod interfaces. If you
+are using our [test cluster], `enp0s9` is the correct value.
+
+```bash
+cat <<EOF | kubectl create -f -
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: macvlan-conf
+spec:
+  config: '{
+      "cniVersion": "0.3.0",
+      "type": "macvlan",
+      "master": "enp0s9",
+      "mode": "bridge",
+      "ipam": {
+        "type": "dhcp"
+      }
+    }'
+EOF
+```
+
+The above definition assumes that DHCP will be used to assign IP addresses to
+the macvlan secondary interfaces. For an alternative solution using the
+[whereabouts] project, please refer to this
+[section](#using-whereabouts-for-ipam).
+
+### Step 4 (optional): Create a macvlan subinterface on each Node
+
+This step is required if you want the Node to be able to communicate with the
+Pods using the secondary network:
+
+```bash
+wget https://raw.githubusercontent.com/vmware-tanzu/antrea/master/docs/cookbooks/multus/resources/macvlan-host-init.yml
+# edit file as needed
+kubectl apply -f macvlan-host-init.yml
+```
+
+If you are using our [test cluster], no edits are required and you can apply the
+file as is. Otherwise, make sure that you replace `enp0s9` with the name of the
+parent network interface.
+
+This manifest will create a DaemonSet that will run a bash script once on every
+Node. It will:
+ * Enable promiscuous mode on the parent interface using `ifconfig`; if using a
+   virtual network for the Nodes, this does not replace enabling promiscuous
```

Comment:
I am not sure I understood this sentence. Would you consider rephrasing?

Reply:
Hopefully it's clearer now, especially with the pointer to the Prerequisites section